packages/salesforcedx-utils-vscode/src/helpers/index.ts

```diff
@@ -6,3 +6,4 @@
  */
 
 export { isNullOrUndefined } from './utils';
+export { extractJsonObject } from './utils';
```

Comment:
Since these are both being exported from the same file, you can combine them into one export statement.